packages/ui/src/theme/css/component/pagination.scss

```diff
@@ -14,64 +14,64 @@
       --amplify-components-pagination-item-container-margin-right
     );
   }
-}
-
-.amplify-pagination__item-button[data-variation='link'] {
-  @extend %item-shared;
-
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  font-weight: initial;
-  color: var(--amplify-components-pagination-button-color);
-  padding-inline-start: var(
-    --amplify-components-pagination-button-padding-inline-start
-  );
-  padding-inline-end: var(
-    --amplify-components-pagination-button-padding-inline-end
-  );
-  transition-property: var(
-    --amplify-components-pagination-button-transition-property
-  );
-  transition-duration: var(
-    --amplify-components-pagination-button-transition-duration
-  );
+  &__item-button--link {
```

Comment:
I feel like we can get rid of `link` without appending a new `xxx--link` class. What do you think?

Reply:
yes good call, link has been removed